### PR TITLE
Feature/nur fixes

### PR DESCRIPTION
**Pull request by chrislombaard**

### CODE REVIEW

File: nurseconnect/views.py [hunk 1]
```diff
@@ -35,8 +35,9 @@ def search(request, results_per_page=7):
     search_query = request.GET.get("q", None)
     page = request.GET.get("p", 1)
     locale = get_locale_code(get_language_from_request(request))
+    search_query = search_query.strip()
 
-    if search_query:
+    if search_query and search_query != "":
```
Comment (phala) [inline nurseconnect/views.py]: that last part, isn't that checked by the first part of the conditional?

File: nurseconnect/views.py [hunk 2]
```diff
@@ -227,6 +228,11 @@ def post(self, request, *args, **kwargs):
                         "old_password",
                         _("The old password is incorrect.")
                     )
+                    messages.success(
```
Comment (phala) [inline nurseconnect/views.py:231]: why do we have this message, as well as the form error?

File: nurseconnect/views.py (hunk 1 above)
Comment (chrislombaard) [reply nurseconnect/views.py]: Oh yes, since it strips the spaces away anyway. cool

File: nurseconnect/views.py (hunk 2 above)
Comment (chrislombaard) [reply nurseconnect/views.py:231]: You're so right.

**Review by phala — APPROVED** [review]

**Review by phala — APPROVED** [review]

**Pull request merged**